envs/public/lookandfeel/setting.json

```diff
@@ -0,0 +1,13 @@
+{
+  "headerBgColor": {
+    "darkMode": "",
+    "lightMode": ""
+  },
+  "loginBgImg": "",
+  "loginTitle": "SCS Harbor Registry",
+  "product": {
+    "name": "SCS Harbor",
+    "logo": "SCS_logo.png",
+    "introduction": "SCS Harbor Registry https://github.com/SovereignCloudStack/k8s-harbor"
```

Comment:
The initial idea was that this section would contain SCS Registry features that we have enabled and are available for registry users, e.g.: 
- we proxy and cache (mirror) several public repositories (docker.io, ghcr.io, quay.io, registry.gitlab.com, registry.k8s.io)
- we do a security scan on a daily bases
- we do a garbage collection on a daily bases